.github/workflows/app-dh-healthchecks.yml

```diff
@@ -38,8 +38,5 @@ jobs:
       - name: 📥 Monorepo install
         uses: ./.github/actions/yarn-nm-install
 
-      - name: Install Playwright
-        run: npx playwright install --with-deps
-
       - name: Run healthchecks
-        run: npx nx run e2e-dh:health
+        run: npx nx run dh-cypress-e2e:e2e:health
```

Comment:
issue: looks like `dh-cypress-e2e` does not exist. Should probably be `e2e-dh:e2e:health`